starfish/starfish.py

```diff
@@ -72,8 +72,18 @@ def version():
 version.no_art = True  # type: ignore
 
 
-@starfish.command()
+@starfish.group()
+def util():
+    """
+    house-keeping commands for the starfish library
+    """
+    pass
+
+@util.command()
 def install_strict_dependencies():
+    """
+    warning! updates different packages in your local installation
```

Comment:
does this need to be in the help text?

Reply:
The “warning”? Nah. But there aren’t many packages that install stuff themselves.

Reply:
Or printed out to the console? Actually the help text for this command is pretty unhelpful.

Reply:
Ha. Fair enough. :smile: But I'm misusing your sign-off to get this into your PR where we can discuss a help message better than `None` and "unhelpful"